app/models/piece.rb

```diff
@@ -58,6 +60,12 @@ def obstructed?(new_x, new_y)
 
   private
 
+  def increment_move
```

Comment:
`app/models/piece.rb:66:5: C: Avoid using increment! because it skips validations.`